### ISSUE TITLE
[Form field] Combobox input does not have properly added label

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3010]
## Current behavior
[Form Field Combobox](https://storybook.sanomalearning.design/?path=/story/form-form-field--combobox) has text `<label>` that is associated with `<sl-combobox>` using `for` and `id` attributes.
`<sl-combobox>` also has `aria-describedby` with hint text.
Both `<label>` and `aria-describedby` shouldn't be added to `sl-combobox` as users won't place their focus on `<sl-combobox>`.
That is why hint is not read now with use of Tab key to navigate page.

<img width="572" height="100" alt="Image" src="https://github.com/user-attachments/assets/e8c718a2-0e18-49bd-9a3f-770340dd69db" />

## Proposed solution
Change value of `for` in `<label>` to `id` value of `<input>`. 
Remove `aria-describedby` from `<sl-combobox>` and add it to `<input>`.

<img width="661" height="323" alt="Image" src="https://github.com/user-attachments/assets/0e47cde6-bdb8-4d27-82c9-659744c24229" />

This way users using Tab to navigate page will get both label name and hint text on the same element.

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
